src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        Random random = new Random();
+        ColorSupplier colorSupplier = new ColorSupplier();
+        colorSupplier.getRandomColor();
+        ball.setColor(colorSupplier.getRandomColor());
+        ball.setNumber(random.nextInt(100));
+        return ball;
+    }
```

Comment:
create a constructor for these fields (color, number) if you need them
random, supplier - same as in color supplier class, should be class fields
100 - class constant, remember, we use CONSTANT_CASE for constants